src/main/java/stsjorbsmod/cards/cull/Discordance.java

```diff
@@ -0,0 +1,68 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Discordance extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Discordance.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+
+
+
+    public Discordance() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.baseDamage = 0;
+        this.baseMagicNumber = magicNumber = 5;
+    }
+
+    @Override
+    public int calculateBonusBaseDamage() {
+        Iterator var2 = AbstractDungeon.player.hand.group.iterator();
```

Comment:
You can use a `Set` and a foreach loop in place of an Iterator to simplify this a bit:

```java
        HashSet<CardType> seen = new HashSet<>();
        for (AbstractCard c : AbstractDungeon.player.hand.group) {
            seen.add(c.type);
        }
```

Alternately, you can use `Stream::distinct`:

```java
int distinctTypes = (int) AbstractDungeon.player.hand.group.stream().map(c -> c.type).distinct().count();
```

Reply:
Updated to match suggestion